### PR TITLE
[TASK] Add yaml lint

### PR DESCRIPTION
**Pull request by lukaszuznanski**
Add yaml lint and lint all files.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,218 +1,191 @@
...
-      run: composer ci:php:lint
-
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Install PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '${{ matrix.php-version }}'
+      - name: Run PHP lint
+        run: 'composer ci:php:lint'
```
Comment (oliverklee) [inline .github/workflows/ci.yml:24]: @lukaszuznanski I find this PR quite hard to read due to the mix of formatting changes and other "real" changes. Would you mind creating a PR for the formatting changes that we can merge first? That would be very helpyful for me. Thanks!

File: .travis.yml [hunk 2]
```diff
@@ -1,66 +1,52 @@
+---
```
Comment (oliverklee) [inline .travis.yml:1]: We've just removed Travis CI support in #56. So this file does not exist anymore.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (oliverklee) [reply .github/workflows/ci.yml:24]: (For the other files, this is fine as there are only formatting changes as far as I can see. Only this file has the mix of formatting changes and new/changed code.)

Comment (lukaszuznanski) [reply .github/workflows/ci.yml:24]: I get your point. I will separate those changes.

File: .travis.yml (hunk 2 above)
Comment (lukaszuznanski) [reply .travis.yml:1]: Ok, fine, I will remove this file

File: .github/workflows/ci.yml (hunk 1 above)
Comment (oliverklee) [reply .github/workflows/ci.yml:24]: Thanks!

**Comment by lukaszuznanski** [comment]
requires #72 merge first

**Comment by lukaszuznanski** [comment]
If it's fine, we should merge this PR as soon as possible, because making further changes to ci.yml would be much easier :)

**Pull request merged**